resources/docker/README.md

```diff
@@ -0,0 +1,169 @@
+# Joinup on Docker
+
+## Requirements
+* Docker
+* docker-compose
+
+## Containers
+To run Joinup in containers, we are using the following images:
+* [php:7.1.24-apache-jessie](https://hub.docker.com/_/php/) which is extended with configuration based on the [docker
+for drupal](https://github.com/docker-library/drupal) and the [fpfis/httpd-php](https://github.com/fpfis/httpd-php)
+images.
+* [pablofelix/sass](https://hub.docker.com/r/pablofelix/sass/)
+* [percona/percona-server:5.6](https://hub.docker.com/r/percona/percona-server/)
+* [tenforce/virtuoso](https://hub.docker.com/r/tenforce/virtuoso/)
+* [selenium/standalone-chrome-debug](https://hub.docker.com/r/selenium/standalone-chrome-debug/)
+* [solr:6](https://hub.docker.com/_/solr/). This container is extended and comes with the search_api_solr configuration
+files installed in /opt/docker-solr/configsets/drupal/conf.
+
+## Getting started
+
+#### Prepare the environment
+From the project root, run `docker-compose up`. This command will download, build and run all necessary containers.
```

Comment:
Only the first part of my remark has been fixed. Let's put all the commands to execute on separate lines instead of inlining them in paragraphs of text.